microsoft-edge/devtools-protocol-chromium/index.md

```diff
@@ -1,33 +1,35 @@
 ---
 description: Update to the Microsoft Edge DevTools Protocol
-title: Microsoft Edge DevTools Protocol Update
+title: Microsoft Edge DevTools Protocol overview
 author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.date: 04/06/2021
 ms.topic: reference
 ms.prod: microsoft-edge
 ---
-# Microsoft Edge (Chromium) DevTools Protocol overview  
+# Microsoft Edge DevTools Protocol overview  
 
-With the shift in the underlying web platform of Microsoft Edge to Chromium, the [Microsoft Edge (EdgeHTML) DevTools Protocol](/archive/microsoft-edge/legacy/developer/devtools-protocol/index) will not be receiving any further updates.  The Microsoft Edge \(Chromium\) DevTools Protocol will match the APIs of the Chrome DevTools Protocol going forward.  
-
-You can find documentation on those domains and methods by referring to the [Chrome DevTools Protocol Viewer](https://chromedevtools.github.io/devtools-protocol/tot).  
+Use the DevTools Protocol to instrument, inspect, debug, and profile browsers including Microsoft Edge.  The Microsoft Edge DevTools Protocol matches the APIs of the Chrome DevTools Protocol.  For reference documentation, navigate to [Chrome DevTools Protocol Viewer](https://chromedevtools.github.io/devtools-protocol/tot).  
 
 > [!NOTE]
+> With the shift in the underlying web platform of Microsoft Edge to Chromium, the [Microsoft Edge (EdgeHTML) DevTools Protocol](/archive/microsoft-edge/legacy/developer/devtools-protocol/index) will not be receiving any further updates.  The Microsoft Edge \(Chromium\) DevTools Protocol will match the APIs of the Chrome DevTools Protocol going forward.  
+> 
 > Any methods that were prefixed with `ms` in the [Microsoft Edge (EdgeHTML) DevTools Protocol](/archive/microsoft-edge/legacy/developer/devtools-protocol/index) are no longer supported in the Microsoft Edge \(Chromium\) DevTools Protocol.  
 
+
 ## Using the DevTools Protocol  
 
-Here's how to attach a custom tooling client to the DevTools Server in Microsoft Edge \(Chromium\).  
+To attach a custom tooling client to the DevTools Server in Microsoft Edge (Chromium):
```

Comment:
Here and elsewhere: are we supposed to be removing mentions of "Chromium" ?

Reply:
There is a work item to create a separate PR to remove instances of "Chromium" and "the new Microsoft Edge" from the docset.  It's assigned to a top writer.

(I am taking notes in GitHub PR UI to help find how to use it, despite quirks.  I'm writing in the Conversation tab right now.
I think I already answered this PR review Comment, maybe in the Files Changed tab.  But this review Comment is now marked as Outdated.   It seems like I have to enter responses twice, sometimes, in a PR.)